packages/experimental/Dropdown/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/dropdown",
-  "version": "0.6.0",
+  "version": "0.6.1",
```

Comment:
NPM says last published is 0.6.0

https://www.npmjs.com/package/@fluentui-react-native/dropdown